### ISSUE TITLE
peertube processing for youtube-dl dont start ( status : waiting )

### ISSUE THREAD

**Issue by sandwitcherz** [Chocobozzz/PeerTube#4568]
Hello, i am on irc, but noone help me ^^ So, importation from a local video works, but when i import a youtube video, the job stuck on "waiting "
```
<html><body>
<!--StartFragment-->

Job: 2 Type: video-import Processed on Nov 25, 2021, 8:07:59 PM Finished on -
--
{   "type": "youtube-dl",   "videoImportId": 2,   "fileExt": ".mp4" }

<!--EndFragment-->
</body>
</html>
```
irc question : ok, i have installed a 3.4.1 peertube whith nginx. Server is okay. I have ffmpeg version 4.1.8-0+deb10u1. i can import mp4 video, thats works, but i can't import youtube videos, i have no errors but when you import, it stucks at " in work "... anyone could help plz?

thanks a lot

**Comment by sandwitcherz** [comment]
`![image](https://user-images.githubusercontent.com/1787239/143497412-fcc16861-91cf-4a5d-a5f1-615fc43dc464.png)`

**Comment by sandwitcherz** [comment]
Error: Cannot guess path of /var/www/peertube/storage/tmp/e3c0c2f8bf58c4086818f50724820ad6c2e6e6b6e1ef0d805deeb95b5c258512-import
    at YoutubeDL.<anonymous> (/var/www/peertube/versions/peertube-v3.4.1/dist/server/helpers/youtube-dl.js:161:19)
    at Generator.next (<anonymous>)
    at fulfilled (/var/www/peertube/versions/peertube-v3.4.1/node_modules/tslib/tslib.js:114:62)

**Comment by SimplyCorbett** [comment]
You're going to need to find the directory youtube-dl is located in, then download:

https://github.com/yt-dlp/yt-dlp/releases/download/2021.11.10.1/yt-dlp

Place it in that directory. Remove youtube-dl, rename yt-dlp to youtube-dl, chown peertube:peertube youtube-dl, chmod +x youtube-dl and restart peertube.

tl;dr youtube-dl is incredibly broken on youtube and is unmaintained.

**Comment by sandwitcherz** [comment]
link is /var/www/peertube/versions/peertube-v3.4.1/node_modules/youtube-dl/bin

Again, you are the Dev of the year ^^ everything works, hope this thread help a lot of people. Thanks a lot, and if i have a question... i know who speak ^^
